docs/docs/sourcing-from-contentstack.md

```diff
@@ -0,0 +1,110 @@
+
+Gatsby, with its powerful UI toolset, serves as the “head” for your websites while Contentstack – being the headless CMS – acts as the “body”. Combining these the two best-in-class tools provide a powerful solution to create modern, enterprise-grade sites and apps.
+
+_**Note:** This guide uses the `gatsby-source-contentstack` plugin to create a Gatsby-powered site._
+
+## Build a Site with Contenstack and Gatsby
+
+### Prerequisites
+
+- [ ] Install Node.js on your system
+- [ ] Install Gatsby CLI: `npm install -g gatsby-cli`
+
+### Step 1: Clone the repo
+
+Clone the repo from GitHub. This contains all the required dependencies.
+
+`git clone https://github.com/contentstack/gatsby-starter-contentstack.git`
+
+### Step 2: Install npm
+
+Go to the ‘gatsby-starter-contentstack’ folder, and run the following:
+
+```cd gatsby-starter-contentstack
+npm install
+```
+
+This downloads the required files and initializes the site.
+
+### Step 3: Update the config file
+
+Add the `api_key`, `delivery_token` and `environment` stack details to the existing `gatsby-source-contentstack` object within the gatsby-config.js file:
+
+```plugins: [{
+    resolve: `gatsby-source-contentstack`,
+    options: {
+      api_key: `api_key`,
+      access_token: `delivery_token`,
+      environment: `environment`
+    },
+  }'
+]
+```
+
+_**Note:** if you wish to preview the pages of your gatsby site from within the Contentstack entry editor, ensure base URLs are set for each environment and language._
+
+### Step 4: Create and publish content
+
+To build a sample home page, perform the following steps in Contentstack:
+
+1. Create a ‘Home’ content type with ‘Title’ and ‘Body’ fields
+1. Create an entry for the ‘Home’ content type
+1. Create a development environment
+1. Publish the ‘Home’ entry to the development environment
+
+### Step 5: Run Gatsby
+
+Navigate to your root directory and run the following command:
+
+`gatsby develop`
+
+After running this, you will be able to view your site at `http://localhost:8000/`. You can run the GraphiQL IDE at `http://localhost:8000/___graphql`. The GraphiQL IDE will help you explore the app's data, including the Contentstack APIs.
+
+Now, you will able to query Contentstack data. Try the query below to get the 'Home' content type data:
+
+```{
+    contentstackHome {
+        title
+        body
+
+}
+```
+
+### Step 6: Create a page in Gatsby
+
+Go to the pages folder inside the src folder, and create a home.js file. Add the code below to it.
+
+```import React from 'react';
+import { graphql } from 'gatsby';
+export default ({ data }) => {
+  return (
+    <div>
+      <h1>{data.contentstackHome.title}</h1>
+    </div>
+
+}
+export const pageQuery = graphql`
+    query HomeQuery {
+      contentstackHome {
+        title
+
+    }`
+```
+
+This will display the title of your home page on `http://localhost:8000/home` :tada:. Likewise, you can query additional fields in your entry.
```

Comment:
Does an emoji written with the slack/rocket style syntax like that render okay? `:tada:`? I'm not sure but if it does great 👍

Reply:
Good point @gillkyle. Went ahead and removed